scripts/test_cases/large_app_with_private_packages_centralised_ngbv_removed_linux.py

```diff
@@ -0,0 +1,28 @@
+"""_summary_
+This script basically runs benchmark test for msbuild
+Returns:
+    None: .....
+"""
+
+import subprocess
+
+EXTRACT_PATH = "sdk"
+WORKING_DIR = "msbuild-performance-test"
+DOTNET_BASE_VERSION_URL_LINUX = "https://download.visualstudio.microsoft.com/download/pr/dc930bff-ef3d-4f6f-8799-6eb60390f5b4/1efee2a8ea0180c94aff8f15eb3af981/dotnet-sdk-6.0.300-linux-x64.tar.gz"
+DOTNET_DAILY_VERSION_URL_LINUX = "https://aka.ms/dotnet/8.0.1xx/daily/dotnet-sdk-linux-x64.tar.gz"
+TEST_SOLUTION_REPO_URL = "https://github.com/marcin-krystianc/TestSolutions.git"
+TEST_REPO_NAME = "TestSolutions"
+TEST_SOLUTION_CASE = "LargeAppWithPrivatePackagesCentralisedSlim"
+TEST_SOLUTION_DIR = "solution"
+SDK_VERSION = "6.0.300"
+SDK_DAILY_VERSION = "8.0.1xx"
```

Comment:
`dotnet --version` gives you back the version - it is not necessary to hardcode these values (Also hardcoding these values may lead to a mismatch between the hardcoded version and the actual version)